website_and_docs/content/documentation/selenium_manager.en.md

```diff
@@ -326,5 +326,23 @@ In this case, the library to be installed is the following:
 sudo apt-get install libatk-bridge2.0-0
 ```
 
+### Using an environment variable for the driver path
+It's possible to use an environment variable to specify the driver path without using Selenium Manager.
+The following environment variables are supported:
+
+* SE_CHROMEDRIVER
+* SE_EDGEDRIVER
+* SE_GECKODRIVER
+* SE_IEDRIVER
+* SE_SAFARIDRIVER
+
+For example, to specify the path to the chromedriver, you can set the `SE_CHROMEDRIVER` environment variable to the path of the chromedriver executable.
```

Comment:
Hi @aguspe ,

Thank you for the PR.
Could you please limit the line length to 100 characters as per the style guidelines
See Line length in https://www.selenium.dev/documentation/about/style/

Thanks,
Sri